content/community.json

```diff
@@ -15,7 +15,7 @@
     },
     "events": {
       "title": "Events",
-      "description": "Learn more about DVC and meet us in person! We're appearing at conferences, meetups, and talks around the world.",
+      "description": "Learn more about DVC and meet our team virtually! We're appearing at conferences, meetups, and talks around the world.",
```

Comment:
make it `or in person`? so that we don't have to change it next time?

Reply:
Good idea!